index.bs

```diff
@@ -885,8 +885,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. [=set/Append=] |authenticator| to |issuedRequests|.
 
-1. [=While=] |issuedRequests| [=list/is not empty=], perform the following actions depending upon
-    |lifetimeTimer| and responses from the authenticators:
+1. [=While=] |lifetimeTimer| has not expired, perform the following actions depending upon |lifetimeTimer| and responses from the
```

Comment:
Why was the clause "issuedRequests not empty" deleted?

Reply:
Because with the previous language the operation would terminate immediately if no authenticator is present at the moment when the timer is started, but step 19 is intended to be run asynchronously as authenticators appear within the timeout. This change is mostly orthogonal to the addition of the privacy considerations section, but it's necessary for the privacy consideration text to make any sense. I also believe we need to do this change for the algorithms themselves to make sense, even disregarding the new privacy consideration.

Reply:
See https://github.com/w3c/webauthn/pull/687#issuecomment-358223418